bin/web.js

```diff
@@ -267,19 +267,30 @@ fs.readFile(path.join(__dirname, '../config/config.json'), 'utf8', (err, data) =
                   key,
                   // return complete object URL
                   uri,
-                  picture
+                  picture,
+                  extra
```

Comment:
```suggestion
                  picture
```

nope, mexer na resposta da api vai quebrar uns apps em frontend e umas intrgrações 😄

Reply:
OK